util/src/main/java/tc/oc/pgm/util/xml/XMLUtils.java

```diff
@@ -1130,4 +1130,15 @@ public static Title.Times parseTitleTimes(Element el, Title.Times def)
 
     return Title.Times.times(fadeIn, stay, fadeOut);
   }
+
+  public static Color parseHexColor(Node node, Color def) throws InvalidXMLException {
```

Comment:
I'd follow the pattern other methods around here do, one parser parses only a non-null node (required), then there's an overload for a potentially-nullable with a fallback default.
In this case it seems like the default is never properly used (always white), i'd simply go for parseHexColor and throw an invalid xml exception if either the node or the node's value are null, and avoid having a default fallback